coding-challenge/src/bank/bank-account.ts

```diff
@@ -10,23 +10,69 @@ export class BankAccount {
         this.accountNumber = uuid();
     }
 
+    /*
+    * Withdraw money from the current account
+    * Params:
+    * withdrawAmount (--number): The amount of money to be withdrawn
+    * 
+    * Returns:
+    * void
+    */
     withdraw(withdrawAmount: number) {
-        if (this.balance >= withdrawAmount) {
+        if (withdrawAmount <= 0){
+            throw new Error('Withdraw amount has to be greater than 0!');
+        }
+        if (this.balance <= withdrawAmount) {
```

Comment:
So you mean if I have 10$ in my bank account, and I want to withdraw 10$ I cannot?